packages/dcos-integration-test/extra/util/python_test_server.py

```diff
@@ -300,7 +305,7 @@ def start_http_server(listen_port):
 
     logging.info("HTTP server is starting, port: "
                  "{}, test-UUID: '{}'".format(listen_port, os.environ[TEST_UUID_VARNAME]))
-    test_server = HTTPServer(('', listen_port), TestHTTPRequestHandler)
+    test_server = ThreadingSimpleServer(('', listen_port), TestHTTPRequestHandler)
```

Comment:
nit: would be nice to have this as a comment in the code so everyone who ever looks at this code in the future knows / can understand the reasoning. Right now they'd have to find the commit via blame then look up the PR comments to find it.